src/ussd_nurse_rapidpro.js

```diff
@@ -33,24 +34,54 @@ go.app = function() {
 
             /*
                 find identity & check subscription using CompanionApp
-
+                use temporary fixture to differentiate non_subscribers and subscribers
+                this will be replaced with method that checks subscriptions
                 will go to state_registered if found,
                 else state_not_registered
             */
-            return self.states.create('state_not_registered');
-
+            if(msisdn === "+27820001003"){ 
+                return self.states.create('state_registered_1');
+            } else {
+                return self.states.create('state_not_registered');
+            }
         });
 
         self.states.add('state_not_registered', function(name){
             return new FreeText(name, {
                 question: ("Welcome to NurseConnect, where you can stay up to date with "+
                 "maternal & child health. Reply '1' to start."),
-                next: 'state_nurse_connect_options'
+                next: 'state_not_registered_menu'
+            });
+        });
+
+        self.states.add('state_registered_1', function(name) {
```

Comment:
I wanted to mention yesterday that I think it would be better to combine this state and the one below it since the jsbox is designed to handle splitting stuff like that. But that is simply a suggestion.

Reply:
could you please help me out with this? I do want to join them somehow and make the code neater

Reply:
@OMosimege I think you should be able to put all the choices in one state and the jsbox utils should automatically handle splitting it. You might not even have to change your tests

Reply:
ahh I see. But I see here that the jsbox utils are part of Seed, in which for this specific project all Seed functions are going to be replaced, so I'll put a note in it and leave it as is for now

Reply:
@OMosimege sorry, I used the wrong term there. Since the ChoiceState is part of the vumigo library I expect that is where it would be handled.